cmd/bosun/conf/rule/modify.go

```diff
@@ -0,0 +1,221 @@
+package rule
```

Comment:
I'm a little shaky on calling this package `rule`. Not sure what would be better though. Maybe `alerts`?